src/xcode/ENA/ENA/Source/SceneDelegate.swift

```diff
@@ -133,22 +157,41 @@ final class SceneDelegate: UIResponder, UIWindowSceneDelegate, RequiresAppDepend
 	}
 
 	private func showHome(animated _: Bool = false) {
+		//FIXME: During the onboarding, if the user decline, the status == Unknown.
+		//After that
 		if exposureManager.preconditions().status == .active {
 			presentHomeVC()
 		} else {
-			log(message: "ExposureManager not activate yet.")
+//			let enManager = ENManager()
+//			enManager.activate { theError in
+//				if let theError = theError {
+//					logError(message: "Cannot activate the  ENManager. The reason is \(theError)")
+//					return
+//				}
+//				self.presentHomeVC()
+//			}
+
 			exposureManager.activate { [weak self] error in
 				if let error = error {
 					// TODO: Error handling, if error occurs, what can we do?
 					logError(message: "Cannot activate the  ENManager. The reason is \(error)")
 					return
 				}
+				//TODO: Set some state
 				self?.presentHomeVC()
 			}
 		}
 	}
 
 	private func presentHomeVC() {
+
+		//FIXME: Move it to other place.
+		let enStateHandler = ENStateHandler(
```

Comment:
current enStateHandler needs to be accessible from SceneDelegate ... Proposal: extract it out of the method